### PR TITLE
Fixturize tests/frame/test_constructors.py

### PR DESCRIPTION
**Pull request by h-vetinari**
One more steps towards #22471. Again, needing to re-add some fixtures that were removed in #24885.

### CODE REVIEW

File: pandas/tests/frame/conftest.py [hunk 1]
```diff
@@ -127,6 +127,22 @@ def timezone_frame():
     return df
 
 
+@pytest.fixture
+def datetime_series():
```
Comment (jreback) [inline pandas/tests/frame/conftest.py]: don't add any more here

**Review by jreback — CHANGES_REQUESTED** [review]
maybe my comments are crossing with your reading of them.

**Comment by h-vetinari** [comment]
@jreback
All green.

PS. As noted in the other threads, this is a `TestData`-"fixture" that's used in several modules.

File: pandas/tests/frame/conftest.py (hunk 1 above)
Comment (jreback) [reply pandas/tests/frame/conftest.py]: not here for now

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/tests/frame/conftest.py (hunk 1 above)
Comment (h-vetinari) [reply pandas/tests/frame/conftest.py]: @jreback 
See above:
> PS. As noted in the other threads, this is a TestData-"fixture" that's used in several modules.

This is used also used in #25627, for example

**Review by h-vetinari — COMMENTED** [review]
I'll reiterate from [another thread](https://github.com/pandas-dev/pandas/pull/25633#pullrequestreview-212612585), that with these fixturization PRs, I will:
* keep the `TestData` "fixtures" in `tests.frame.conftest`
* any new fixtures will stay in their respective modules
* the effort for #22471 is strictly translation, no breakup of tests
* [reorg of fixtures from `TestData` into submodules to happen later]

**Comment by h-vetinari** [comment]
Just for context, these fixtures had already been there (since #22236), and were only recently removed in #24885, even though I noted that there was an ongoing (is somewhat stalled) translation effort.

Can we leave the re-org for later and not re-litigate the fixtures that we already agreed to use in #22236?

File: pandas/tests/frame/conftest.py (hunk 1 above)
Comment (jreback) [reply pandas/tests/frame/conftest.py]: i don't think these fixtures useful; if you really really want them put them in the actual test file for now

**Review by jreback — CHANGES_REQUESTED** [review]

**Comment by jreback** [comment]
pls merge master

**Comment by h-vetinari** [comment]
re `datetime_series`:
> @jreback: i don't think these fixtures useful; if you really really want them put them in the actual test file for now

I agree with you, but currently `tests.frame.common.TestData.ts1` is still used in several places, and will have to be on the level of `tests.frame.conftest` - another PR that needs the same fixture is #25627

**Comment by jreback** [comment]
closing as stale

**Comment by h-vetinari** [comment]
@jreback

I had merged master like you asked, this PR is not stale. Please reopen.

**Comment by h-vetinari** [comment]
@jreback @gfyoung @simonjayhawkins 
Can you please reopen this PR? Now that #25627 is in, this will be easier.

**Comment by h-vetinari** [comment]
Ok, merged and cleaned. Got rid of the `datetime_series` and `datetime_series_short` fixture in the same way as @jreback in [this commit](https://github.com/pandas-dev/pandas/pull/25627/commits/8afc817bd292804791a60f72cee32f0a58f24525) of #25627.

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
lgtm.

**Comment by jreback** [comment]
thanks @h-vetinari

**Pull request merged**